contrib/devtools/update-translations.py

```diff
@@ -159,6 +161,11 @@ def postprocess_translations(reduce_diff_hacks=False):
                 for translation in translations:
                     if translation is None:
                         continue
+                    
```

Comment:
hi, delete the whitespace to make the build pass. The linter is failing.

Reply:
it's in line 164

Reply:
Can also install ```clang-format``` and run https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy to fix any whitespaces before pushing

Reply:
done..thank you